distribution/lib/Standard/Base/0.0.0-dev/src/Internal/Data_Read_Helpers.enso

```diff
@@ -0,0 +1,46 @@
+private
+
+import project.Data.Text.Text
+import project.Data.Vector.Vector
+import project.Enso_Cloud.Data_Link
+import project.Errors.Problem_Behavior.Problem_Behavior
+import project.Metadata.Display
+import project.Metadata.Widget
+import project.Network.HTTP.HTTP
+import project.Network.HTTP.HTTP_Method.HTTP_Method
+import project.Network.URI.URI
+from project.Data.Boolean import Boolean, False, True
+from project.Data.Text.Extensions import all
+from project.Data import Raw_Response
+from project.Metadata.Choice import Option
+from project.Metadata.Widget import Single_Choice
+from project.System.File_Format import Auto_Detect, format_types
+
+
+## PRIVATE
+looks_like_uri path:Text -> Boolean =
+    (path.starts_with "http://") || (path.starts_with "https://")
```

Comment:
should this be case insensitive?

Reply:
I just moved existing code but I think you are right, it may be better to check case insensitively - I will amend that.